base/metrics/field_trial_unittest.cc

```diff
@@ -925,7 +925,7 @@ class FieldTrialListTest : public ::testing::Test {
 
 // TODO(b/298237462): Try to enable |FieldTrialList|.
 // TODO(b/316198056): Determine if tests should be enabled.
-#if !defined(USE_HACKY_COBALT_CHANGES)
+#if !defined(COBALT_PENDING_CLEAN_UP)
 #if !BUILDFLAG(IS_IOS)
```

Comment:
Places like those should probably eventually become `#if !BUILDFLAG(IS_IOS) && !BUILDFLAG(IS_STARBOARD)`

No need to do it in the PR, i'm just leaving comments in a few places as to how i think these could be cleaned up.